noetic/distribution.yaml

```diff
@@ -2483,6 +2483,24 @@ repositories:
       url: https://github.com/ethz-adrl/ifopt.git
       version: master
     status: maintained
+  ifxsensor:
+    doc:
+      type: git
+      url: https://github.com/Yang-IFX/ifxsensor.git
+      version: master
+    release:
+      packages:
+      - ifx_bgt60_device
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/Yang-IFX/ifxsensor-release.git
+      version: 0.0.1-2
+    source:
+      test_pull_requests: true
```

Comment:
@Yang-IFX have you followed [all the steps](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to enable pull request testing?

Reply:
Hi @hidmic, thanks very much for your comments. I just double checked my repo setting. Do miss the "Collaborators" of "ros-pull-request-builder". Long vacation lost things. Thank you.

Reply:
Hi @hidmic , thanks for your last comment. I added "ros-pull-request-builder" from my repo: https://github.com/Yang-IFX/ifxsensor/settings/access . It's been several days. Still on the "pending invite" status. Do you have any idea about this? Thank you.

Reply:
@nuclearsandwich @clalancette are one of you getting these notifications? Dirk used to get the notifications and accept them.

Reply:
I don't get notifications by email. I've spiked out a script that signs in and automatically accepts these periodically but have never put it into production.

Reply:
I just got a notification.  I've now logged in and accepted this one.

Reply:
Thank you very much.